### PR TITLE
Added user statistics

### PR DESCRIPTION
**Pull request by Cheshiriks**
Added user statistics

Снимок232

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt [hunk 1]
```diff
@@ -55,24 +53,27 @@ val cardUser: FC<IndexViewProps> = FC { props ->
...
-            .decodeFromJsonString()
-
-        setCountVulnerability(countVuln)
+        props.userInfo?.name?.let {
+            val organizationsNew: List<OrganizationDto> = get(
+                "$apiUrl/organizations/get/list-by-user-name?userName=$it",
+                jsonHeaders,
+                loadingHandler = ::loadingHandler,
+            )
+                .decodeFromJsonString()
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt:64]: Probably `unsafeCast` is needed (maybe not)

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt [hunk 2]
```diff
@@ -216,5 +217,57 @@ val cardUser: FC<IndexViewProps> = FC { props ->
...
+                    p {
+                        +"Vulnerabilities: "
+                    }
+                }
+
+                div {
+                    className = ClassName("col-3")
+                    p {
+                        Link {
+                            to = "/${FrontendRoutes.PROFILE}/${props.userInfo?.name}"
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt:245]: What if `userInfo` is null

**Review by sanyavertolet — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt:64]: Did you mean ```unsafeMap``` ?
I don't think it's needed here.

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt (hunk 2 above)
Comment (Cheshiriks) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt]: Fixed it

Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt]: Exactly, this is what happening on current master

**Review by akuleshov7 — APPROVED** [review]

**Pull request merged**